examples/dev/README.md

```diff
@@ -0,0 +1,12 @@
+# Example deployment
+
+## Local testing
+* replace the Terraform state S3 bucket fields in the command below as appropriate
+* for local testing, duplicate `terraform.tfvars.example`, rename without the `.example` suffix
+* enter details appropriate to your organization and save
+
+```bash
+terraform init -backend-config=bucket=TERRAFORM_STATE_BUCKET -backend-config=key=TERRAFORM_STATE_KEY -backend-config=region=TERRAFORM_STATE_REGION
+terraform workspace new dev
```

Comment:
```suggestion
```

Reply:
updated to show use of Terraform workspace is optional, and to explain the circumstance in which it's useful - if a single Terraform state bucket is used for multiple environments.